src/PokemonTrainer.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class PokemonTrainer extends Person {
+
+    // Constructor //
+    public PokemonTrainer(String name) {
+        super(name);
+
+    }
+
+    // Getters and setters //
+    public Pokemon getBlastoise() {
```

Comment:
Het was beter geweest om eerst de variabelen boven aan de klasse te definieren en die in deze getter te returnen, net zoals je in de PokemonGymowner class hebt gedaan.

Reply:
Op deze manier kun je namelijk niet de Pokemon aanpassen, bijvoorbeeld de hp van een pokemon verminderen als deze aangevallen wordt in een battle.